src/components/Search.js

```diff
@@ -14,20 +15,22 @@ export class Search extends Component {
       }
 
     render() {
-        const { activeSearchBy } = this.props;
-
+        const { activeSearchBy, searchStr } = this.props;
+        
         return (
             <section className="search-section">
-                <h4 className="white-text">FIND YOUR</h4>
-                <input type="text" placeholder="Enter title" className="white-text search-title"
+                <h4 className="white-text">FIND YOUR MOVIE</h4>
+                <input type="text" placeholder="Enter title/genre" className="white-text search-title"
                         onChange={this.handleChange}></input>
                 <div className="search-container">
                     <div className="search-selection">
                         <label className="white-text">SEARCH BY</label>
                         <button id="title-btn" className={activeSearchBy === 'title' ? 'white-text' : 'white-text nonselected-color'} onClick={this.handleSearchClick}>TITLE</button>
                         <button id="genre-btn" className={activeSearchBy === 'genres' ? 'white-text' : 'white-text nonselected-color'} onClick={this.handleSearchClick}>GENRE</button>
                     </div>
-                    <button className="white-text" onClick={this.props.fetchMovies}>SEARCH</button>
+                    <Link to={'/search/Search ' + searchStr}>
```

Comment:
`/search/${encodeURIComponent(searchStr)}`
Let's discuss it during the meeting.